client/src/components/Graph/Plot.vue

```diff
@@ -39,7 +39,7 @@ export default {
   methods: {
     generatePlot() {
       debug('plot being built');
-      buildPlot(this.plotElementId, this.plotData);
+      buildPlot(this.plotElementId, this.$store.state.plot);
     },
   },
   beforeDestroy() {
```

Comment:
`plotData` on this component is no longer needed if it isn't being used here 😄

Reply:
This is needed. It doesn't update the plot when things change if I use store.